pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[project]
+name = "Bootstrap-Flask"
+description = "Bootstrap 4 & 5 helper for your Flask projects."
+readme = "README.md"
+license = { text = "MIT" }
+authors = [{ name = "Grey Li", email = "withlihui@gmail.com" }]
+classifiers = [
+    'Development Status :: 5 - Production/Stable',
+    'Environment :: Web Environment',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Programming Language :: Python :: 3.12',
+    'Framework :: Flask',
+    'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+]
+dependencies = [
+    "Flask",
+    "WTForms",
+]
+dynamic = ["version"]
+
```

Comment:
Based on the CI error output, adding a  `packages` or `py_modules` field is necessary.

Reply:
This is caused by tool.setuptools.dynamic .